README.md

```diff
@@ -3,35 +3,79 @@
 
 # Kodi Addon checker
 
-This tool checks the Kodi repo for best practices and creates problem and warning reports
+This tool checks the Kodi repo for best practices and creates problem and warning reports.
```

Comment:
the tool checks not only some best practices but mainly our Addon-rules/submission guidelines (https://kodi.wiki/view/Add-on_rules). The way it is written also implies you are creating 2 different reports (one for problem and another for warning). Use maybe: produces a report containing found problems and warnings.